x-pack/plugins/security_solution/public/detections/components/rules/description_step/helpers.tsx

```diff
@@ -201,6 +199,38 @@ export const buildUnorderedListArrayDescription = (
   return [];
 };
 
+export const buildHighlightedFieldsOverrideDescription = (
+  label: string,
+  values: string[]
+): ListItems[] => {
+  if (isEmpty(values)) {
+    return [];
+  }
+  const description = (
+    <EuiFlexGroup responsive={false} gutterSize="xs" wrap>
+      {values.map((val: string) =>
+        isEmpty(val) ? null : (
+          <EuiFlexItem grow={false} key={`${label}-${val}`}>
+            <EuiBadgeWrap
+              data-test-subj="customHighlightedFieldsStringArrayDescriptionBadgeItem"
+              color="hollow"
+            >
+              {val}
+            </EuiBadgeWrap>
+          </EuiFlexItem>
+        )
+      )}
+    </EuiFlexGroup>
+  );
+
+  return [
+    {
+      title: label,
+      description,
+    },
+  ];
+};
```

Comment:
@nikitaindik We'll need to add support for this new rule field to the components you're working on. We can do it in a separate PR once @yctercero merges this one.